src/homepage/howTo/dkim.html

```diff
@@ -0,0 +1,178 @@
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+<!-- Main -->
+<div id="main">
+
+    <!-- Introduction -->
+    <section id="intro" class="main special">
+        <div class="">
+            <div class="content align-left">
+                <header class="major">
+                    <h1><b>What is a DKIM Record?</b></h1>
+                </header>
+
+                <p>
+                  DKIM (DomainKeys Identified Mail) is an email security standard designed to make sure messages aren’t altered in transit between the sending and recipient servers.                   
+                </p>
+
+                <p>
+                  It uses public-key cryptography to sign email with a private key as it leaves a sending server. 
+                  Recipient servers then use a public key published to a domain’s DNS to verify the source of the message, and that the body of the message hasn’t changed during transit. 
+                </p>
+
+                <p>
+                  Once the signature is verified with the public key by the recipient server, the message passes DKIM and is considered authentic.
+                </p>
+
+                <p>
+                  The process of setting up DKIM can be split into the following steps:
+                </p>
+
+                <ul>
+                    <li>Choose a DKIM selector.</li>
+                    <li>Generate a public-private key pair.</li>
+                    <li>Publish the selector and public key by creating a DKIM TXT record.</li>
+                    <li>Attach the token to each outgoing email.</li>
+                </ul>
+
+                <p>
+                  Before we begin, you might wonder what is a DKIM selector?
+                </p>
+                <p>
+                  In short, a selector is specified as an attribute for a DKIM signature and is recorded in the DKIM-Signature header field. 
+                  A selector can be anything you want, such as a word, number, or a string of letters and numbers.
+              </p>
+              <p>
+                For example, if you choose <code>james3</code> for your selector, the DKIM record name would become <code>james3._domainkey</code>
+              </p>
+
+
+                <header class="major">
+                    <h1><b>Generate RSA Key Pair for DKIM</b></h1>
+                </header>
+
+                <p>
+                  You can use tools such as <code>openssl</code> or <code>ssh-keygen</code> to generate RSA keys.
+                </p>
+                <p>
+                 Please note that 1024 bit DKIM is still the standard. If you want to feel safer with 2048-bit RSA, check with DNS provider and see what length of DKIM key is supported because they need to match.
```

Comment:
```suggestion
                 Please note that 1024 bit DKIM is still the standard. If you want to feel safer with 2048-bit RSA, check with your DNS provider and see what length of DKIM key is supported because they need to match.
```